src/tbb/CMakeLists.txt

```diff
@@ -126,6 +126,25 @@ target_link_libraries(tbb
     ${TBB_COMMON_LINK_LIBS}
 )
 
+# Strip debug symbols into a separate .dbg file
+if(TBB_LINUX_SEPARATE_DBG)
+    if(NOT CMAKE_BUILD_TYPE STREQUAL "release")
```

Comment:
Should we strip it for `debug` configuration?

Reply:
The debug symbols will not be stripped in any default configuration. If user set the BUILD_TYPE='debug' or relwithdebinfo and explicitly set the TBB_LINUX_SEPARATE_DBG switch, then the symbols get stripped to a different file and linked to binary. Whereas when BUILD_TYPE is 'release' the .dbg file has only the linkage information and hence creation of .dbg is avoided.